daedalus/src/Daedalus/Crypto.purs

```diff
@@ -1,8 +1,39 @@
 module Daedalus.Crypto where
 
+import Prelude
+
+import Data.ArrayBuffer.Types (Uint8Array)
+import Data.String (replaceAll, Pattern (..), Replacement (..))
+
 import Control.Monad.Eff (Eff)
 
 foreign import data CRYPTO :: !
 
 foreign import isValidMnemonic :: String -> Boolean
 foreign import generateMnemonic :: forall eff. Eff (crypto :: CRYPTO | eff) String
+
```

Comment:
Looks like all code modifications in this module shuld be reverted, because they corrspond to base64-url